client/src/components/toyProblems/ToyProblems.jsx

```diff
@@ -34,7 +35,8 @@ class ToyProblems extends Component {
   }
 
   getCohortsList() {
-    getAllCohorts().then(result => {
+    const cohortsQuery = useDB ? getAllCohorts : getAllCohortsNoDb;
```

Comment:
I'll review the source code later but I wonder if we can hide this in the model code rather than here in the component.  That is, `getAllCohorts` uses a database or does not as specified in the queries rather than here.

What do you think?

Reply:
I agree. I am working towards this, but didn't complete it. Ideally, we should have 1 flag to control all components (similar to a feature flag).